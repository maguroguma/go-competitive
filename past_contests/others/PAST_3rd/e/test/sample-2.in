30 10 20
11 13
30 14
6 4
7 23
30 8
17 4
6 23
24 18
26 25
9 3
18 4 36 46 28 16 34 19 37 23 25 7 24 16 17 41 24 38 6 29 10 33 38 25 47 8 13 8 42 40
2 1 9
1 8
1 9
2 20 24
2 26 18
1 20
1 26
2 24 31
1 4
2 21 27
1 25
1 29
2 10 14
2 2 19
2 15 36
2 28 6
2 13 5
1 12
1 11
2 14 43
