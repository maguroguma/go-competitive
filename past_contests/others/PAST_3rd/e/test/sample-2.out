18
19
37
29
8
24
18
25
46
10
18
42
23
4
17
8
24
7
25
16
